package.json

```diff
@@ -27,7 +27,7 @@
     "eslint-staged:src": "STRICT=1 eslint -c src/.eslintrc.js",
     "eslint-staged:cypress": "STRICT=1 eslint -c cypress/.eslintrc.js",
     "eslint:strict": "STRICT=1 eslint -c src/.eslintrc.js 'src/**/*.ts?(x)' && eslint -c cypress/.eslintrc.js 'cypress/**/*.ts'",
-    "notify-email": "env-cmd -e prod -r env/.cmdrc.json node cli-scripts/notify-email.js",
+    "notify-email": "env-cmd -e prod -r env/.cmdrc.json ./email.sh",
```

Comment:
Is this for local deploys? If so, should it invoke email.sh with bash? Or should email.sh start with a shebang specifying bash?